packages/core/src/config/spec.ts

```diff
@@ -0,0 +1,31 @@
+import type { PluginStyleguideConfig } from './types';
+
+const spec: PluginStyleguideConfig<'built-in'> = {
```

Comment:
I believe there should be a page describing this ruleset.

Reply:
I will add file but I think we will need to populate it later as I am missing rules description documentation.
cc: @JLekawa